backend/app/routers/profile.py

```
@@ -43,3 +43,16 @@ async def update_profile(
     db.commit()
     db.refresh(db_user)
     return {"message": "Profile updated successfully"}
+
+
+@router.put("/become-artisan")
+async def become_artisan(
```

Ruff (0.8.2)

Missing return type annotation for public function `become_artisan`

(ANN201)